pkg/server/delta/v3/server.go

```diff
@@ -120,7 +120,18 @@ func (s *server) processDelta(str stream.DeltaStream, reqCh <-chan *discovery.De
 			watch := watches.deltaWatches[typ]
 			watch.nonce = nonce
 
-			watch.state.SetResourceVersions(resp.GetNextVersionMap())
+			if !watch.state.IsWildcard() {
```

Comment:
Can we drop a comment in here about why non-wildcard has to be treated differently? Or a link to some docs would also be nice. It'll help me updating the changelog down the line.

Reply:
sure, will do that.

Reply:
done